.github/workflows/ci_cron.yml

```diff
@@ -328,3 +328,36 @@ jobs:
       - name: Annotate errors
         if: ${{ failure() }}
         uses: ./.github/actions/annotate
+
+  gem_notifications:
+    runs-on: ubuntu-20.04
+    env:
+      gems:
+        "activerecord 
+        bunny
+        dalli
+        delayed_job
+        excon
+        http
+        httpclient
+        mongo
+        puma
+        sidekiq
+        sinatra
+        tilt
+        rack
+        rails
+        rake 
+        redis
+        resque
+        unicorn"
```

Comment:
1. Putting one gem on each line is a cool way to do it. 👍 
2. I didn't realize earlier that the "Check for outdated gems" step already has its own `env:` section. The "gems" env var can be moved down there and this global-to-all-steps `env:` list can be deleted.